stdlib/csv/from.go

```diff
@@ -169,6 +178,13 @@ func (c *CSVSource) Run(ctx context.Context) {
 		if err != nil {
 			goto FINISH
 		}
+		if results.More() {
+			err = errors.New(
+				codes.FailedPrecondition,
+				"csv.from() should only return 1 result",
```

Comment:
nit, I think this error message can read slightly better. Instead of saying it should only return 1 result, perhaps state it can only parse,read one result, this way its clear that if you get this error the fix is to provide only a single result to `csv.from()`.